src/main/java/frc/robot/subsystems/shooter/ShooterConstants.java

```diff
@@ -0,0 +1,73 @@
+package frc.robot.subsystems.shooter;
+
+public class ShooterConstants {
+
+  private static final String CONSTRUCTOR_EXCEPTION = "constant class";
+
+  private ShooterConstants() {
+    throw new IllegalStateException(CONSTRUCTOR_EXCEPTION);
+  }
+
+  public static final boolean DEBUGGING = false;
+  public static final boolean TESTING = true;
+  public static final String SUBSYSTEM_NAME = "Shooter";
+
+  public static final int TOP_SHOOTER_MOTOR_ID = 27;
+  public static final int BOTTOM_SHOOTER_MOTOR_ID = 28;
+  public static final int ANGLE_MOTOR_ID = 29;
+  public static final int ANGLE_ENCODER_ID = 9;
+
+  public static final int SETPOINTCOUNT = 5;
+
+  // Shooter Velocity PID Constants
+  public static final double SHOOT_KP = 40.0;
+  public static final double SHOOT_KI = 0.0;
+  public static final double SHOOT_KD = 0.0;
+  public static final double SHOOT_KS = 14.0;
+  public static final double SHOOT_PID_PEAK_OUTPUT = 1.0;
+
+  // Shooter Rotation PID Constants
+  public static final double ROTATION_KP = 40.0;
+  public static final double ROTATION_KI = 0.0;
+  public static final double ROTATION_KD = 1.0;
+  public static final double ROTATION_KS = 1.0;
+  public static final double ROTATION_KG = 0.08;
+  public static final double ROTATION_KV = 1.0;
+  public static final double ROTATION_KA = 0.01;
+  public static final double ROTATION_EXPO_KV = 24.0;
```

Comment:
we learned with the chain-driven arm that we have to set achievable velocity targets; this is not achievable with the gear ratio of the arm as the stator's max speed is 100 rps; so, let's try a value of 80 to start